--- origsrc/clisp-2.49.50/modules/syscalls/configure	2017-03-19 17:24:42.000000000 -0400
+++ src/clisp-2.49.50/modules/syscalls/configure	2017-03-20 16:10:04.628900900 -0400
@@ -4983,7 +4983,12 @@ fi
 done
 
 if test "$ac_cv_header_shlobj_h" = yes ; then
-   LIBS=${LIBS}' -luser32 -lole32 -loleaut32 -luuid -lversion';
+   case "$ac_cv_build" in
+   *-cygwin)
+     LIBS=${LIBS}' -luser32 -lole32 -loleaut32 -L/usr/lib/w32api -luuid -lversion';;
+   *)
+     LIBS=${LIBS}' -luser32 -lole32 -loleaut32 -luuid -lversion';;
+   esac
 fi
 # Check whether --enable-largefile was given.
 if test "${enable_largefile+set}" = set; then :
